v4/source/system.yaml

```diff
@@ -48,6 +48,15 @@
             $ref: "#/definitions/StatusOK"
         '403':
           $ref: '#/responses/Forbidden'
+      x-code-samples:
+        - lang: 'Go'
+          source: |
+            import "github.com/mattermost/platform/model"
+
+            Client := model.NewAPIv4Client("https://your-mattermost-url.com")
+            Client.Login("email@domain.com", "Password1")
+
+            ok, err := Client.DatabaseRecycle()
```

Comment:
The `err` should be `resp` here and in the other examples too

Reply:
done